docs/index.md

```diff
@@ -1227,7 +1227,7 @@ Environment variables and default values:
 | `--google-auth-token`     | `GOOGLE_AUTH_TOKEN`   | -                                    |
 | `--google-scopes`         | `GOOGLE_SCOPES`       | `devstorage.read_only,logging.write` |
 | `--google-disk-size`      | `GOOGLE_DISK_SIZE`    | `10`                                 |
-| `--google-disk-type`      | `GOOGLE_DISK_TYPE`    | `pd-statdard`                        |
+| `--google-disk-type`      | `GOOGLE_DISK_TYPE`    | `pd-standard`                        |
```

Comment:
I wasn't sure if this was a typo or an option I didn't understand, but I had a look at where this flag is defined ([drivers\google\google.go](https://github.com/docker/machine/blob/master/drivers/google/google.go), line 86) and it does just seem to be a typo.